src/components/AboutUs/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { withStyles } from 'material-ui'
+import { Grid } from 'material-ui'
```

Comment:
Can be done in single import statement: `import { withStyles, Grid } from 'material-ui'`